controllers/storagecluster/cephobjectstores_test.go

```diff
@@ -48,7 +51,8 @@ func assertCephObjectStores(t *testing.T, reconciler StorageClusterReconciler, c
 	err = reconciler.Client.Get(context.TODO(), request.NamespacedName, actualCos)
 	// for any cloud platform, 'cephobjectstore' should not be created
 	// 'Get' should have thrown an error
-	if avoidObjectStore(reconciler.platform.platform) {
+	isavoidObjectStore, _, _ := reconciler.avoidObjectStore(cr)
```

Comment:
For all instances like this, fix the capitalization:
```suggestion
	isAvoidObjectStore, _, _ := reconciler.avoidObjectStore(cr)
```